packages/cli/templates/defaults/app/route-templates/html.tsx

```diff
@@ -240,66 +244,83 @@ const getMethod = (value: string | undefined) => {
   }
 };
 
-export const action = async ({ request, context }: ActionFunctionArgs) => {
-  const formData = await request.formData();
+export const action = async ({
+  request,
+  context,
+}: ActionFunctionArgs): Promise<
+  { success: true } | { success: false; errors: string[] }
+> => {
+  try {
+    const formData = await request.formData();
 
-  const formId = getFormId(formData);
-  if (formId === undefined) {
-    // We're throwing rather than returning { success: false }
-    // because this isn't supposed to happen normally: bug or malicious user
-    throw json("Form not found", { status: 404 });
-  }
+    const formId = formData.get(formIdFieldName);
 
-  const formProperties = formsProperties.get(formId);
+    if (formId == null || typeof formId !== "string") {
+      throw new Error("No form id in FormData");
+    }
 
-  // form properties are not defined when defaults are used
-  const { action, method } = formProperties ?? {};
+    const formBotValue = formData.get(formBotFieldName);
 
-  if (contactEmail === undefined) {
-    return { success: false };
-  }
+    if (formBotValue == null || typeof formBotValue !== "string") {
+      throw new Error("Form bot field not found");
+    }
 
-  // wrapped in try/catch just in cases new URL() throws
-  // (should not happen)
-  let pageUrl: URL;
-  try {
-    pageUrl = new URL(request.url);
+    const submitTime = parseInt(formBotValue, 16);
+    // 5 minutes
```

Comment:
whats the idea about 5 minutes, please write full sentences, I am guessing its the time between user clicking the submit button and server receiving the data?